demos/basics/loading-saving-neo4j.ipynb

```diff
@@ -1063,10 +1063,7 @@
    "source": [
```

Comment:
<p>same as prev notebook, should "is simpler" be "are simpler"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>